website/src/components/IIGO/IIGO.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import logo from '../../assets/logo/logo512.png';
+import styles from './IIGO.module.css';
+
+function IIGO() {
+  return (
+    <div className={styles.root}>
+      <img src={logo} className={styles.appLogo} alt="logo" />
+      <p className={styles.text}>
+        Self-Organising Multi-Agent Systems 2020
+        </p>
+    </div>
+  );
+}
+
+export default IIGO;
```

Comment:
every org has a logo?

Reply:
oh, read the desc again. I think just a title shall do. no need for the logo and the copy-pasta css

Reply:
A better placeholder than a blank box. Can be changed later. Visualisations other than Resources and roles aren't even set in stone yet.